_schedule/talks/2022-10-18-09-45-t0-come-on-in-the-waters-fine-making-python.md

```diff
@@ -4,12 +4,13 @@ category: talks
 date: 2022-10-18 09:45:00-07:00
 difficulty: All
 end_date: 2022-10-18 10:30:00-07:00
+group: talks
 layout: session-details
 link: null
-permalink: /talks/keynote-come-on-in-waters-fine/
+permalink: /talks/come-on-in-the-waters-fine-making-python/
```

Comment:
We need to update the blog post that links to this talk

Reply:
Fixed!

Reply:
Reverted back to the OG.